packages/matchbox/src/components/ScreenReaderOnly/ScreenReaderOnly.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './ScreenReaderOnly.module.scss';
+
+const ScreenReaderOnly = ({ children }) => (
+  <span className={styles.ScreenReaderOnly}>
+    {children}
+  </span>
+);
+
+ScreenReaderOnly.propTypes = {
+  children: PropTypes.node.isRequired
```

Comment:
Any reason anything other than a string would be used here?

Reply:
You *might* have other elements inside of this guy - so it could be a `<label></label>` element or even a `<legend></legend>`. You might even have hidden headings inside these as well.